composer.json

```diff
@@ -60,7 +60,7 @@
         "doctrine/doctrine-cache-bundle": "~1.0",
         "mmoreram/simple-doctrine-mapping": "~0.1",
 
-        "sebastian/money": "~1.3",
+        "sebastian/money": "1.5.*",
```

Comment:
I am talking about BC Break Promise. That means that you can use `~1.5` with any fear of BC Break.